pkgs/by-name/nz/nzportable/fteqw.nix

```
@@ -0,0 +1,176 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  cmake,
+  ninja,
+  pkg-config,
+  makeWrapper,
+  zip,
+  gettext,
+
+  libpng,
+  zlib,
+  gnutls,
+  libGL,
+  xorg,
+  alsa-lib,
+  libjpeg,
+  libogg,
+  libvorbis,
+  libopus,
+  dbus,
+  fontconfig,
+  SDL2,
+  bullet,
+  openexr,
+  sqlite,
+  addOpenGLRunpath,
```

```suggestion
  addDriverRunpath,
```
see ofborg eval error.

Besides this, LGTM.

---

🤦🏼‍♀️ I keep forgetting that it's called something else now, maybe we should add an eval warning whenever someone uses it